adapters/repos/db/batch_integration_test.go

```diff
@@ -398,7 +398,9 @@ func testBatchImportGeoObjects(repo *DB) func(t *testing.T) {
 					Schema: map[string]interface{}{
 						"location": randGeoCoordinates(),
 					},
-					Vector: []float32{0.123, 0.234, 0.345}, // does not matter for this test
+					Additional: &models.AdditionalProperties{
+						Vector: []float32{0.123, 0.234, 0.345}, // does not matter for this test
```

Comment:
Vector should not be part of AdditionalProperties in REST, as outlined in https://docs.google.com/document/d/1o_77mX0cMyGPDB6DQYeWRKwy4ZsLP7rZYAxr4YKzNlg/edit#heading=h.ptkfthm2plht

If it is required in there (for GraphQL?) then we should double it into `models.Object` as well.